webserver/views/api.py

```diff
@@ -129,6 +131,17 @@ def get_listens(user_id):
     }})
 
 
+def _parse_int_arg(name, default=None):
+        value = request.args.get(name)
+        if value:
+            try:
+                return int(value)
+            except ValueError:
+                raise BadRequest("Invalid %s argument." % name)
```

Comment:
I think the error message should include `value` as well because not everyone reading the message is a developer or wants to capture network requests to find out what value some scrobbler is sending. MB itself also includes invalid values in error messages.